script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -1021,33 +1029,34 @@ Divine providence that could not be opposed.
 [sha:f4bc5fabb7dce4273dacc72b4a3aaa11d2bf2118]{
 -- Page 619, Offset 25914.
 -- わかり合える筈もない生き物だった。
-A creature beyond any form of mutual understanding.
+A creature unable to form any sort of mutual understanding.
 }
 [sha:11c1ca9043063d07b7d48344b2d2eb1976cf2025]{
 -- Page 620, Offset 25915.
 -- 嵐のように渦巻く瞳。
-Eyes that swirl like a storm.
+Eyes, that swirl like a storm.
 }
 [sha:2d1afb9545bff72a559bfd5166c7db00fb36c83c]{
 -- Page 620, Offset 25916.
 -- 山脈のようにとがった歯。
-Teeth spiked like a mountain range.
+Teeth, spiked, like a mountain range.
 }
 [sha:160baf04c6ac12c5b758da5aa609b15fa2085dc6]{
 -- Page 620, Offset 25917.
 -- 意思の疎通など出来ようのない、<瀑布|ばくふ>じみた力の<撓|たわ>み。
-A curving mass of power like a waterfall with no room for communication.
+// double check, I think I kinda messed this one up.
```

Comment:
I think the original is correct here 

Maybe something like: A curved mass of force like a waterfall; allowing no room for mutual understanding.

I don't really have anything great to offer as an alternative, but I do think the original's meaning is right